<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Horizontal_NetworkClass</class>
 <widget class="QMainWindow" name="Horizontal_NetworkClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1686</width>
    <height>943</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Horizontal_Network</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>389</x>
      <y>0</y>
      <width>70</width>
      <height>841</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>870</y>
      <width>1691</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>20</y>
      <width>421</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="titleLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Business</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="businessLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="controlButton">
       <property name="text">
        <string>Control</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="ControlLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="measureButton">
       <property name="text">
        <string>Measure</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="MeasureLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="angleButton">
       <property name="text">
        <string>Angle</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="AngleLineEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="lineButton">
       <property name="text">
        <string>Line</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="LineLineEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="openFileButton_5">
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="outputLineEdit"/>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>10</y>
      <width>611</width>
      <height>831</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>Measure Point</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="MeasureTable"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string>Line Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="LineTable"/>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>850</y>
      <width>1651</width>
      <height>24</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>611</width>
      <height>831</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Control Point</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="ControlTable"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>Angle Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="AngleTable"/>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>401</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1686</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Horizontal_Network.qrc"/>
 </resources>
 <connections/>
</ui>
